index.bs

```diff
@@ -353,12 +353,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Authentication</dfn>
 :: The [=ceremony=] where a user, and the user's computing device(s) (containing at least one [=authenticator=]) work in
-    concert to cryptographically prove to an [=[RP]=] that the user controls the [=credential private key=] associated with a
+    concert to cryptographically prove to a [=[RP]=] that the user controls the [=credential private key=] associated with a
```

Comment:
This is incorrect.  The a/an prefix operates phonetically - not by spelling.  Because "RP" sounds like it starts with a vowel sound, "an" is correct.

Reply:
Yes, that was my thinking as well, but `[=[RP]=]` expands to `[=Relying Party=]`. If these were `[=RP=]` (note the missing inner brackets) I would agree with you. :)

Reply:
You're right!  Then this is ready to go then.